src/main/java/org/sunbird/cb/hubservices/profile/handler/ProfileUtils.java

```diff
@@ -33,6 +33,7 @@ public static List<String> getUserDefaultFields() {
 		userFields.add(Constants.PROFILE_DETAILS_PROFESSIOANAL_DETAILS);
 		userFields.add(Constants.PROFILE_DETAILS_EMPLOYMENT_DETAILS);
 		userFields.add(Constants.PROFILE_DETAILS_PERSONAL_DETAILS);
+		userFields.add(Constants.PROFILE_DETAILS);
```

Comment:
You should use different one... 
this suppose to be with new variable in constant..
public static final String PROFILE_DETAILS_VERIFIED_KARMAYOGI = "profileDetails.verifiedKarmayogi";
not the entire profileDetails object